server/src/main/java/org/elasticsearch/index/mapper/FieldMapper.java

```diff
@@ -268,7 +268,7 @@ public CopyTo copyTo() {
      * update if dynamic mappings modified the mappings, or {@code null} if
      * mappings were not modified.
      */
-    public Mapper parse(ParseContext context) throws IOException {
+    public void parse(ParseContext context) throws IOException {
```

Comment:
I think it makes sense to also update the method comment according to the change.

Reply:
Oops, thanks!